File: VB/CustomWizardExample/Wizard/Presenters/ChoosePersonsDataSourceTypePageEx.vb
Imports CustomWizardExample.Wizard.Views
Imports DevExpress.DataAccess.Wizard

Namespace CustomWizardExample.Wizard.Presenters
	Public Class ChoosePersonsDataSourceTypePageEx
		Inherits ChoosePersonsDataSourceTypePage

        Public Sub New(ByVal view As IChoosePersonsDataSourcePageView)
            MyBase.New(view)
        End Sub

		Public Overrides ReadOnly Property MoveNextEnabled() As Boolean
			Get
				Return False
			End Get
		End Property

		Public Overrides ReadOnly Property FinishEnabled() As Boolean
			Get
				Return MyBase.MoveNextEnabled
			End Get
		End Property

	End Class
End Namespace

File: CS/CustomWizardExample/Wizard/Presenters/ChoosePersonsDataSourceTypePageEx.cs
using CustomWizardExample.Wizard.Views;
using DevExpress.DataAccess.Wizard;

namespace CustomWizardExample.Wizard.Presenters {
    public class ChoosePersonsDataSourceTypePageEx : ChoosePersonsDataSourceTypePage {
        public ChoosePersonsDataSourceTypePageEx(IChoosePersonsDataSourcePageView view, IDataSourceWizardClient client) : base(view, client) { }

        public override bool MoveNextEnabled { get { return false; } }

        public override bool FinishEnabled { get { return base.MoveNextEnabled; } }

    }
}
